CHANGELOG.md

```diff
@@ -1,5 +1,33 @@
 # Changelog
 
+## 1.13.0
+
+- ### Features
+
+  - **Add support for text blob bindings - [xortive], [pull/1543], [issue/483]**
+    
+    Wrangler now supports text blobs! text blobs are values to use in your workers, but read from a file instead of a string in your TOML.
```

Comment:
`Text blobs are values to use in your workers, but are read from a file instead of a string in your TOML.